samples/grpc-server/pom.xml

```diff
@@ -30,22 +30,15 @@
 	<properties>
 		<java.version>17</java.version>
 		<spring-javaformat-maven-plugin.version>0.0.39</spring-javaformat-maven-plugin.version>
-		<protobuf-java.version>3.25.2</protobuf-java.version>
-		<grpc.version>1.63.0</grpc.version>
+		<protobuf-java.version>3.25.5</protobuf-java.version>
```

Comment:
I don't think these are doing anything actually as they are not managed by SB dependencies